IMPEDANCE_FILTER:
  M: [1, 1, 1, 0.0833333, 0.0833333, 0.0833333]
  B: [6.32455532, 6.32455532, 6.32455532, 0.5270462767, 0.5270462767, 0.5270462767]
  K: [10, 10, 10, 0.83333333 , 0.83333333, 0.83333333]
MRAC:
  REFERENCE_MODEL:
    OMEGA: [3.16227766, 3.16227766, 3.16227766, 1.054092553, 1.054092553, 1.054092553]
    ZETA: [1, 1, 1, 1, 1, 1]
  WEIGHTING_FACTORS:
    Wp: [2.0, 2.0, 5.0, 2.0, 2.0, 0.0]
    Wd: [1.0, 1.0, 1.0, 0.8, 0.8, 0.0]
  INTEGRAL_ADAPTATION_GAINS:
    a1: [0.2, 0.2, 0.025, 0.04, 0.04, 0.0]
    b1: [0.1, 0.1, 0.00006, 0.00003, 0.00003, 0.0]
    c1: [0.1, 0.1, 0.00000002, 0.000000005, 0.000000005, 0.0]
  PROPORTIONAL_ADAPTATION_GAINS:
    a2: [0.0, 0.0, 0.0025, 0.004, 0.004, 0.0]
    b2: [0.0, 0.0, 0.00006, 0.00003, 0.00003, 0.0]
    c2: [0.0, 0.0, 0.00000002, 0.000000005, 0.000000005, 0.0]
  SIGMA_MODIFICATION_GAINS:
    sigma1: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
    sigma2: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
    sigma3: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  INITIAL_GAINS:
    Kp: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
    Kd: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]